src/ChartCreator/Options/Options.js

```diff
@@ -0,0 +1,55 @@
+// @flow
+import React, { useState, useEffect } from "react";
+import Grid from "@material-ui/core/Grid";
+import Aggregate from "./Aggregate";
+import OrderBy from "./OrderBy";
+import AxesStartingAtZero from "./AxesStartingAtZero";
+
+const dateTypes = ["date", "dateTime", "time"];
+
+const Options = ({
+  orderByColumns,
+  orderBy,
+  setOrderBy,
+  aggregateOperation,
+  setAggregateOperation,
+  unitGroupBy,
+  setUnitGroupBy,
+  valueGroupBy,
+  setValueGroupBy,
+  abscissType,
+  axesStartingAtZero,
+  setAxesStartingAtZero
+}) => {
+  const [isTimeType, setIsTimeType] = useState(false);
+
+  useEffect(() => {
+    setIsTimeType(dateTypes.includes(abscissType));
+  }, [abscissType]);
+
```

Comment:
delete useState and useEffect and make a function returning `dateTypes.includes(abscissType)` to be used below would be better?

Reply:
Imo, faster to check a boolean than check a value inside a tab. Both are working anyway